CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 * (types) [#19759](https://github.com/cosmos/cosmos-sdk/pull/19759) Align SignerExtractionAdapter in PriorityNonceMempool Remove.
 * (client) [#19870](https://github.com/cosmos/cosmos-sdk/pull/19870) Add new query command `wait-tx`. Alias `event-query-tx-for` to `wait-tx` for backward compatibility.
 * (genutil) [#19971](https://github.com/cosmos/cosmos-sdk/pull/19971) Allow manually setting the consensus key type in genesis
+* (client) [#20266](https://github.com/cosmos/cosmos-sdk/pull/20266) Ability to override short description in AutoCLI generated top-level commands.
```

Comment:
Should be under client/v2

Reply:
https://github.com/cosmos/cosmos-sdk/pull/20266/commits/3a200d59b780c6d6cef8107c23a654b54e629bbb